docs/manual/constant-value-checker.tex

```diff
@@ -251,6 +251,18 @@ \section{Warnings\label{value-checker-warnings}}
 
 \end{itemize}
 \end{sloppypar}
+
+\section{Overflow\label{value-checker-overflow}}
+\begin{sloppypar}
+The Constant Value Checker normally takes Java's normal overflow rules into account when
+computing the possible values of expressions. However, the overflow rules can be turned off
+for range annotations (\<@IntRange>, for instance) if desired. This is achieved by passing the
+\code{-AignoreRangeOverflow} flag to the Value Checker. Ignoring overflow is useful for some checkers
```

Comment:
How is it useful?  How does a programmer make the decision of whether to use the flag or not?